### PR TITLE
[RFC] Add code of conduct

### PR DESCRIPTION
**Pull request by gouttegd**
This PR tentatively addresses #3175 by proposing a code of conduct adapted from the [Contributor Covenant](https://www.contributor-covenant.org) code.

This requires community approval (not just a single maintainer’s approval) before merging.

Maintainers are also invited to use that PR to update the `docs/team.md` file as needed to ensure that the list of team members is correct and contains contact information (at least a GitHub handle), since that list is supposed to be the way to contact maintainers when the CoC needs to be enforced.

### CODE REVIEW

**Comment by aleixpuigb** [comment]
It looks great! Let me know if you want me to approve it once it has community approval.

File: CODE_OF_CONDUCT.md
Comment (nicolevasilevsky) [inline CODE_OF_CONDUCT.md:5]:
```suggestion
We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community.
```
I suggest to move this to the top, so it starts on a more positive note

File: CODE_OF_CONDUCT.md
Comment (nicolevasilevsky) [inline CODE_OF_CONDUCT.md:30]:
```suggestion
* Disrespectful behavior, mannerisms, or attacks
* Use of disrespectful, hostile, confrontational, or alienating language
```
I suggest adding a couple more bullet points at the beginning

File: CODE_OF_CONDUCT.md
Comment (nicolevasilevsky) [inline CODE_OF_CONDUCT.md:63]: should there be more details about how to report a complaint, like a specific point of contact? (or multiple points of contact?)

**Comment by nicolevasilevsky** [comment]
This is great @gouttegd, thanks so much for putting this together. I added a few comments and a couple suggested changes.
Thank you! 🌷

File: CODE_OF_CONDUCT.md
Comment (gouttegd) [reply CODE_OF_CONDUCT.md:63]: I don’t think so.

Having a single point of contact would mean either having a single person to be the contact point, or having a contact point that can be read by several persons. Both ways are problematic:

a) If there is a single person to contact: What if that person is not available for any reason (prolonged leave for example)? More importantly, what if that person is precisely the person who allegedly violated the CoC and whose behaviour needs to be reported?

b) If there is a contact point that can be read by several persons: again, what if one of the persons with access to the messages is the alleged perpetrator?

Pointing to the list of the project members and letting the person who needs to report a violation choose the person they are sending the report to is, I believe, both the easiest and the safest option for everyone.

File: CODE_OF_CONDUCT.md
Comment (gouttegd) [reply CODE_OF_CONDUCT.md:5]: No objection to that.

File: CODE_OF_CONDUCT.md
Comment (gouttegd) [reply CODE_OF_CONDUCT.md:30]: No objection to the second point (“use of disrespectful, … language“).

Unsure about the first. Contrary to the other examples given below, this one feels vague and too open to interpretation. What is an “attack”? What is a “disrespectful behavior”?

“You closed my issue without addressing it. That’s disrespectful!”

Comment (nicolevasilevsky) [reply CODE_OF_CONDUCT.md:30]: Fair enough! Feel free to disregard these suggestions as you see fit. :) (I was looking at the OBO Foundry code of conduct for specific language that they had used - https://obofoundry.org/docs/COC.html, but I agree, this can be subjective.)

File: docs/team.md
Comment (matentzn) [inline docs/team.md:18]: 🤣

**Review by matentzn — APPROVED** [review]
BIG APPROVE

File: docs/team.md
Comment (gouttegd) [reply docs/team.md:18]: I strive for accurate descriptions. 😎

**Comment by gouttegd** [comment]
The decision was made in the Uberon call of April 17th to adopt this CoC.

@AvolaAmg , @Caroline-99, @JABelfiore Please check that the informations about you are accurate and let me know if you have any changes to make (e.g. regarding the FTEs you can allocate to Uberon; for now I’ve put you at zero, as almost all other editors).

File: docs/team.md
Comment (aleixpuigb) [inline docs/team.md:15]:
```suggestion
| Alida Avola (@AvolaAmg) | BICAN/HMBA | 0 | [Editors](https://github.com/orgs/obophenotype/teams/uberon-cl-curators) | Ontology Editor | Brain/nervous system, new term requests for BICAN |
```

**Review by aleixpuigb — APPROVED** [review]
Everything looks good. Alida, Caroline and Jasmine will need to be added to https://github.com/orgs/obophenotype/teams/uberon-cl-curators as well.

**Comment by gouttegd** [comment]
Thanks for the reminder @aleixpuigb . I’ve invited them now.

**Comment by aleixpuigb** [comment]
Thank you!

**Review by matentzn — APPROVED** [review]
Beautiful!

**Pull request merged**
